NConfiguration/Serialization/SimpleTypes/Parsing/TimeSpanParser.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace NConfiguration.Serialization.SimpleTypes.Parsing
+{
+    public class TimeSpanParser
```

Comment:
for new files, you should also use tabs for indenting

Reply:
Fixed